src/psyclone/psyir/tools/dependency_tools.py

```diff
@@ -734,21 +699,22 @@ def can_loop_be_parallelised(self, loop,
                                  signatures_to_ignore=None):
         # pylint: disable=too-many-branches,too-many-locals
         '''This function analyses a loop in the PsyIR to see if
-        it can be safely parallelised over the specified variable.
+        it can be safely parallelised.
 
         :param loop: the loop node to be analysed.
         :type loop: :py:class:`psyclone.psyir.nodes.Loop`
-        :param bool only_nested_loops: if True, a loop must have an inner\
-                                       loop in order to be considered\
+        :param bool only_nested_loops: if True, a loop must have an inner
```

Comment:
I believe this parameter can now be removed as well. I can't see it used anywhere.

Reply:
Oops, done.